x-pack/plugins/apm/server/lib/helpers/create_es_client/call_async_with_debug.ts

```diff
@@ -81,17 +81,26 @@ export async function callAsyncWithDebug<T>({
   return res;
 }
 
-export const getDebugBody = (
-  params: Record<string, any>,
-  requestType: string
-) => {
+export const getDebugBody = ({
+  params,
+  requestType,
+  operationName,
+}: {
+  params: Record<string, any>;
+  requestType: string;
+  operationName?: string;
+}) => {
+  const operationLine = operationName ? `${operationName}\n` : '';
```

Comment:
Isn't `operationName` required?

Reply:
for certain calls, it isn't (yet), eg indexing some docs with the internal client

Reply:
Okay. If possible I think we should make it required. But not important for this PR.

Reply:
I'll do it in this PR as well, shouldn't be that big of a change.